src/bin/pg_autoctl/watch_colspecs.h

```diff
@@ -0,0 +1,173 @@
+			{ COLUMN_TYPE_REPORTED_STATE, "Reported State", 0 },
+			{ COLUMN_TYPE_ASSIGNED_STATE, "Assigned State", 0 },
+			{ COLUMN_TYPE_LAST, "", 0 }
+		}
+	},
+	{
+		"very terse",
+		0,
+		{
+			{ COLUMN_TYPE_ID, "Id", 0 },
+			{ COLUMN_TYPE_CONN_REPORT_LAG, "Lag(R)", 0 },
+			{ COLUMN_TYPE_REPORTED_STATE, "Reported State", 0 },
+			{ COLUMN_TYPE_ASSIGNED_STATE, "Assigned State", 0 },
+			{ COLUMN_TYPE_LAST, "", 0 }
+		}
+	},
+	{
+		"terse",
+		0,
+		{
+			{ COLUMN_TYPE_ID, "Id", 0 },
+			{ COLUMN_TYPE_CONN_REPORT_LAG, "Lag(R)", 0 },
+			{ COLUMN_TYPE_CONN_HEALTH, "Connection", 0 },
+			{ COLUMN_TYPE_REPORTED_STATE, "Reported State", 0 },
+			{ COLUMN_TYPE_ASSIGNED_STATE, "Assigned State", 0 },
+			{ COLUMN_TYPE_LAST, "", 0 }
+		}
+	},
+	{
+		"standard",
+		0,
+		{
+			{ COLUMN_TYPE_NAME, "Name", 0 },
+			{ COLUMN_TYPE_ID, "Id", 0 },
+			{ COLUMN_TYPE_CONN_REPORT_LAG, "Lag(R)", 0 },
+			{ COLUMN_TYPE_CONN_HEALTH, "Connection", 0 },
+			{ COLUMN_TYPE_REPORTED_STATE, "Reported State", 0 },
+			{ COLUMN_TYPE_ASSIGNED_STATE, "Assigned State", 0 },
+			{ COLUMN_TYPE_LAST, "", 0 }
+		}
+	},
+	{
+		"semi verbose",
+		0,
+		{
+			{ COLUMN_TYPE_NAME, "Name", 0 },
+			{ COLUMN_TYPE_ID, "Id", 0 },
+			{ COLUMN_TYPE_CONN_REPORT_LAG, "Lag(R)", 0 },
+			{ COLUMN_TYPE_CONN_HEALTH_LAG, "Lag(H)", 0 },
+			{ COLUMN_TYPE_CONN_HEALTH, "Connection", 0 },
+			{ COLUMN_TYPE_REPORTED_STATE, "Reported State", 0 },
+			{ COLUMN_TYPE_ASSIGNED_STATE, "Assigned State", 0 },
+			{ COLUMN_TYPE_LAST, "", 0 }
+		}
+	},
+	{
+		"verbose",
+		0,
+		{
+			{ COLUMN_TYPE_NAME, "Name", 0 },
+			{ COLUMN_TYPE_ID, "Node", 0 },
+			{ COLUMN_TYPE_REPLICATION_QUORUM, "Quorum", 0 },
+			{ COLUMN_TYPE_CANDIDATE_PRIORITY, "Priority", 0 },
+			{ COLUMN_TYPE_CONN_REPORT_LAG, "Lag(R)", 0 },
+			{ COLUMN_TYPE_CONN_HEALTH_LAG, "Lag(H)", 0 },
+			{ COLUMN_TYPE_CONN_HEALTH, "Connection", 0 },
+			{ COLUMN_TYPE_REPORTED_STATE, "Reported State", 0 },
+			{ COLUMN_TYPE_ASSIGNED_STATE, "Assigned State", 0 },
+			{ COLUMN_TYPE_LAST, "", 0 }
+		}
+	},
+	{
+		"almost full",
+		0,
+		{
+			{ COLUMN_TYPE_NAME, "Name", 0 },
+			{ COLUMN_TYPE_ID, "Node", 0 },
+			{ COLUMN_TYPE_REPLICATION_QUORUM, "Quorum", 0 },
+			{ COLUMN_TYPE_CANDIDATE_PRIORITY, "Priority", 0 },
+			{ COLUMN_TYPE_TLI_LSN, "TLI: LSN", 0 },
+			{ COLUMN_TYPE_CONN_REPORT_LAG, "Lag(R)", 0 },
+			{ COLUMN_TYPE_CONN_HEALTH_LAG, "Lag(H)", 0 },
+			{ COLUMN_TYPE_CONN_HEALTH, "Connection", 0 },
+			{ COLUMN_TYPE_REPORTED_STATE, "Reported State", 0 },
+			{ COLUMN_TYPE_ASSIGNED_STATE, "Assigned State", 0 },
+			{ COLUMN_TYPE_LAST, "", 0 }
+		}
+	},
+	{
+		"full",
+		0,
+		{
+			{ COLUMN_TYPE_NAME, "Name", 0 },
+			{ COLUMN_TYPE_ID, "Node", 0 },
+			{ COLUMN_TYPE_REPLICATION_QUORUM, "Quorum", 0 },
+			{ COLUMN_TYPE_CANDIDATE_PRIORITY, "Priority", 0 },
+			{ COLUMN_TYPE_HOST_PORT, "Host:Port", 0 },
+			{ COLUMN_TYPE_TLI_LSN, "TLI: LSN", 0 },
+			{ COLUMN_TYPE_CONN_REPORT_LAG, "Report-Lag", 0 },
+			{ COLUMN_TYPE_CONN_HEALTH_LAG, "Health-Lag", 0 },
```

Comment:
Maybe use Last Report and Last Health instead of Lag, because that was considered confusing by some people.
```suggestion
			{ COLUMN_TYPE_CONN_REPORT_LAG, "Report-Lag", 0 },
			{ COLUMN_TYPE_CONN_HEALTH_LAG, "Health-Lag", 0 },
```

Reply:
Yeah I changed that to “Last Check” and “Last Report” in the long forms, and made the health check column move just before the connection column, then we have the last report column, and only then we have the reported/assigned states. The short forms are now just “Check” and “Report” and it might be explicit enough...